src/client/components/Logout/Logout.style.css

```diff
@@ -0,0 +1,114 @@
+/*Log button on front end styling */
+.logbutton {
```

Comment:
A "log" is somewhere you can list events, such as in a log book or something like that, so I think the name `logbutton` is confusing. Use `logout-button` instead. Or maybe you don't need any class at all if you re-use the button component as discussed above.